robobreizh/GraspConfig best_grasp
---
std_msgs/Bool result